jenkins-scripts/sdformat-default-devel-windows-amd64.bat

```diff
@@ -9,7 +9,13 @@ set IGN_CLEAN_WORKSPACE=true
 set DEPEN_PKGS="libxml2 tinyxml2"
 
 for /f %%i in ('python "%SCRIPT_DIR%\tools\detect_cmake_major_version.py" "%WORKSPACE%\%VCS_DIRECTORY%\CMakeLists.txt"') do set SDFORMAT_MAJOR_VERSION=%%i
-if %SDFORMAT_MAJOR_VERSION% GEQ 6 (
+
+if %SDFORMAT_MAJOR_VERSION% GEQ 10 (
+  :: This needs to be migrated to DSL to get multi-major versions correctly

```

Comment:
I'm not sure what this comment means. I think we only check which distro to use in the dsl; I believe we always check the major version in scripts. we certainly know the major version for the release-branch jobs, but not the `ci-pr_any`

Reply:
I agree, not sure where it comes from, removed.